Apps/Playground/UWP/App.cpp

```diff
@@ -359,6 +360,9 @@ void App::RestartRuntime(Windows::Foundation::Rect bounds)
 {
     Uninitialize();
 
+    Babylon::DebugTrace::EnableDebugTrace(true);
+    Babylon::DebugTrace::SetTraceOutput([](const char* trace) { OutputDebugStringA(trace); });
```

Comment:
Need a newline probably
```suggestion
    Babylon::DebugTrace::SetTraceOutput([](const char* trace) { OutputDebugStringA(trace); OutputDebugStringA("\n"); });
```

Reply:
Fixed